### PR TITLE
Updated strings and added ref number to selection

### PR DESCRIPTION
**Pull request by jaredlhf**
added message reply to users and slaves upon completion of task (simp and normal), added ref id in selection of task to complete

### CODE REVIEW

**Review by jaredlhf — COMMENTED** [review]
ignore this

File: src/leaderboard.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function getLeaderboardRow(userID) {
   var lastColumn = rangeData.getLastColumn();
   var searchRange = leaderboard_sheet.getRange(2, 1, lastRow - 1, lastColumn);
   var rangeValues = searchRange.getValues();
-  var result = "Leaderboards" + '\n';
+  var result = "Leaderboards 👑".bold() + '\n\n';
```
Comment (vimuthm) [inline src/leaderboard.js]: -s

File: src/main.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ function doPost(e) {
         }
       } else if (text === '/take_request') {
         if (processRequest(userId) === false) {
-          sendText(chatID, 'You have no requests to take');
+          sendText(chatID, 'There are no requests left');
```
Comment (vimuthm) [inline src/main.js]: "There are no active requests to take up!"

File: src/profile.js [hunk 3]
```diff
@@ -51,11 +52,11 @@ function getProfileKeyboard(keyboardNumber) {
   var firstKeyboard = [
           [
             {
-              text: 'Check Own Requests',
+              text: 'My Requests Made',
```
Comment (vimuthm) [inline src/profile.js:55]: scrap made

File: src/profile.js [hunk 4]
```diff
@@ -79,7 +80,7 @@ function getProfileKeyboard(keyboardNumber) {
               callback_data: 'toggleProfile-' + 1,
             },
             {
-              text: 'Check Own Requests',
+              text: 'My Requests Made',
```
Comment (vimuthm) [inline src/profile.js:83]: scrap made

File: src/view.js [hunk 5]
```diff
@@ -46,7 +46,7 @@ function updateView(userID, data, message_id) {
   ]
   var str = getTestView(index);
   if (str === 'Showing Active Requests\n\n') {
-    updateText(userID, message_id, "That's all the active requests!", { inline_keyboard: finalkeyboard });
+    updateText(userID, message_id, "There are no requests left!", { inline_keyboard: finalkeyboard });
```
Comment (vimuthm) [inline src/view.js]: "There are no active requests to take up!"

**Review by vimuthm — CHANGES_REQUESTED** [review]

**Pull request merged**